### DISCUSSION TITLE
An unknown error happened, caused the Phrase 2 sealing interrupt

### DISCUSSION THREAD

**Discussion by verynohair** [filecoin-project/lotus#8188] [Storage Provider]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [X] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
lotus version 1.15.0-dev+mainnet+git.ff10e0eaf
lotus-miner version 1.15.0-dev+mainnet+git.ff10e0eaf
```

### Describe the Bug

An error caused the P2 interrupt. Shows below:

Segmentation fault      (core dumped) nohup `BELLMAN_CUSTOM_GPU="GeForce GTX 1080:2560"` lotus-worker run --listen=192.168.0.5:3456 --addpiece=false --precommit1=false --unseal=false --precommit2=true --commit=true --parallel-fetch-limit=2 > /var/tmp/worker.out 2>&1

### Logging Information

```text
2022-02-24T00:55:43.014 INFO filcrypto::proofs::api > seal_pre_commit_phase2: start
2022-02-24T00:55:43.014 INFO filecoin_proofs::api > validate_cache_for_precommit_phase2:start
2022-02-24T00:55:43.015 INFO filecoin_proofs::api > validate_cache_for_precommit_phase2:finish
2022-02-24T00:55:43.015 INFO filecoin_proofs::api::seal > seal_pre_commit_phase2:start
2022-02-24T00:55:43.015 INFO storage_proofs_porep::stacked::vanilla::proof > replicate_phase2
2022-02-24T00:55:43.015 INFO storage_proofs_porep::stacked::vanilla::proof > Building trees [65535 descriptors max available]
2022-02-24T00:55:43.015 INFO storage_proofs_porep::stacked::vanilla::proof > generating tree c using the GPU
2022-02-24T00:55:43.015 INFO storage_proofs_porep::stacked::vanilla::proof > Building column hashes
2022-02-24T00:55:43.036 INFO neptune::proteus::program > Using kernel on OpenCL.
```

**Comment by Reiers** [comment]
**Hi @verynohair** 
Since theres no errors in the log provided, and Im not able to reproce the issue on a 1080 with the exact environment provided.  I will have to move this issue to Discussions for help and troubleshooting. 

I will recommend to stay on the latest stable tag and not run master if this is production. 
https://github.com/filecoin-project/lotus/releases/tag/v1.14.2

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new update from the author.
